src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -494,12 +495,13 @@ amplify add custom
 Next, install the AppSync dependencies for your custom resource:
 ```bash
 cd amplify/backend/custom/MyCustomResolvers
-npm i @aws-cdk/aws-appsync@~1.124.0
+npm i @aws-cdk/aws-appsync@~1.172.0
```

Comment:
this should point to some `~2.x` version. I.e.`@aws-cdk/aws-appsync-alpha@~2.53.0-alpha.0`

Reply:
We can make this change once we release the CDK 2 change.